shortcircuit-api/src/main/java/com/powsybl/shortcircuit/Fault.java

```diff
@@ -0,0 +1,60 @@
+/**
+ * Copyright (c) 2022, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.shortcircuit;
+
+/**
+ * Interface to describe the characteristics of the fault to be simulated.
+ * Used for elementary short-circuit calculation only.
+ *
+ * @author Anne Tilloy <anne.tilloy at rte-france.com>
+ */
+public interface Fault {
+
+    // VERSION = 1.0
+    String VERSION = "1.0";
+
+    // How the fault impedance is connected to the network.
+    enum ConnectionType {
+        SERIES,
+        PARALLEL,
+    }
+
+    // What kind of fault is simulated
+    enum FaultType {
+        THREE_PHASE,
+        TWO_PHASE,
+        SINGLE_PHASE,
+    }
+
+    //TODO : add the numbers of the phase for two and single phase
+
+    // The equipment or bus id where the fault is simulated.
+    String getId();
+
+    default double getProportionalLocation() {
+        return Double.NaN;
+    }
+
+    // Characteristics of the short-circuit.
+    double getR();
+
+    double getX();
+
+    default ConnectionType getConnectionType() {
+        return ConnectionType.SERIES;
+    }
+
+    default FaultType getFaultType() {
+        return FaultType.THREE_PHASE;
+    }
+
+    // Whether the result should indicate a limit violation
+    boolean withLimitViolations();
```

Comment:
**About the question of having those options in the fault or not:**

As discussed, I think it's not very good to mix computation options within an object which has a business signification outside the context of this computation, the same way we don't include computation options in `Contingency`.

Another option would be to include it in parameters, for example as a list of identifiers.
But this does not look good either, because parameters should not depend too much on the network of one particular computation.

Maybe a third and better alternative, and more consistent with security analysis API, is to take inspiration from the `StateMonitor` mechanism : we could provide an additional argument to the computation to specify for which faults we want detailed results